NicolaeChristof/Descend-Into-Darkness
# Feature rope

Hey guys so I just finished the core mechanic of our game, the Rope Swing. Hope you guys enjoy.

How to test:
1. move around the scene, you should see pictures of nathan appear, these represent the grab points
2. use left mouse click to attach yourself to a nearby grab point
3. use arrow keys while attached to move around
4. left click again to detach from grab point
5. click and hold left click while near a grab point to swing, releasing left click will detach from grab point
Try to make it to the top of the scene
